docs/src/content/docs/it/guides/customization.mdx

```diff
@@ -311,6 +317,7 @@ Per utilizzare Google Fonts, segui la [guida alla configurazione di Fontsource](
 
 #### Configura i file dei caratteri locali
 
+<Steps>
```

Comment:
```suggestion
<Steps>

```

Just checked locally by running `pnpm format`, this is missing an empty line so that Prettier does not break the formatting.